src/lib/app.ts

```diff
@@ -139,7 +139,13 @@ export default async function getApp(
         rbacMiddleware(config, stores, services.accessService),
     );
 
-    app.use('/api/admin', maintenanceMiddleware(config));
+    app.use(
+        `${baseUriPath}/api/admin`,
+        conditionalMiddleware(
```

Comment:
I'd move it to the controller so that we get openapi spec

Reply:
This is just middleware for blocking requests, not adding any route.